ui/jobs/buff_tracker.ts

```diff
@@ -307,9 +320,11 @@ export class BuffTracker {
         cooldown: 60,
       },
       litany: {
+        cooldownAbility: kAbility.BattleLitany,
         gainEffect: EffectId.BattleLitany,
         loseEffect: EffectId.BattleLitany,
         useEffectDuration: true,
+        partyonly: true,
```

Comment:
Sorry for the naming nit, but can you make this `partyOnly` since it's two words?